lte/cloud/configs/nprobe.yml

```diff
@@ -9,3 +9,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+updateIntervalSecs: 60
+backoffIntervalSecs: 360
```

Comment:
- Switch the names of these to snake_case to match conventions of other `yml` files
- Add some documentation comments on what these intervals are used for